pelican/plugins/_utils.py

```diff
@@ -107,3 +108,23 @@ def load_plugins(settings):
         plugins = list(namespace_plugins.values())
 
     return plugins
+
+
+def get_plugin_name(plugin):
```

Comment:
Over here: https://github.com/getpelican/pelican/blob/8033162ba4393db60791b201fb100d1be0f04431/pelican/tests/test_plugins.py#L85 things seem to check for a `NAME` attribute on a plugin. Do we need to do this too?

Reply:
No, that was something I added to the tests. It's not a required interface for plugins. I suppose we can switch those to the `get_plugin_name` as well.